lib/proof.rb

```diff
@@ -0,0 +1,55 @@
+class Proof < FakeRecord
```

Comment:
why isn't this a subclass of ActiveRecord? Shouldn't proofs be stored in the database?